addons/mass_mailing/static/src/js/mailing_mailing_view_form_full_width.js

```diff
@@ -176,8 +62,8 @@ const MassMailingFullWidthFormController = FormController.extend({
      * @private
      * @param {JQuery} ev.data.$iframe
      */
-    _onIframeUpdated(ev) {
-        const $iframe = ev.data.$iframe;
+    _updateIframe() {
+        const $iframe = $('iframe.wysiwyg_iframe:visible, iframe.o_readonly');
```

Comment:
try to remove jquery

Reply:
This is pointless to try to use something else than jquery without adapting everywhere `$iframe` is used in this function. Which is not the role of this PR.